index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Quiz App</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <h1>Quiz App</h1>
+    <div class="container">
+      <div class="question-card">
+        <div class="inner-card">
+          <img src="images/211-bookmark.png" alt="bookmark" class="card-bookmark">
+          <h2 class="question">
+            What property flips the axes in flexbox?
+          </h2>
+          <button>Show Answer</button>
+        </div>
+        <div class="badges">
+          <p class="html-badge">#html</p>
+          <p class="flexbox-badge">#flexbox</p>
+          <p class="css-badge">#css</p>
+        </div>
+      </div>
+    </div>
+    <div class="navbar">
+      <img src="images/001-home.png" alt="">
+      <img src="images/211-bookmark.png" alt="">
+      <img src="images/profile-1.png" alt="" class="profile">
+    </div>
```

Comment:
See if you can find semantic markup to improve its readability for screen readers and SEO.